Serializer/JMSSerializerAdapter.php

```diff
@@ -96,6 +96,11 @@ private function convertContext(Context $context, $direction)
         if (null !== $context->getSerializeNull()) {
             $jmsContext->setSerializeNull($context->getSerializeNull());
         }
+        if (is_array($strategies = $context->getExclusionStrategies())) {
```

Comment:
Sure can. I will update the PR when I get home today. The `is_array` should move to the `setExclusionStrategies()` method instead then. I suppose the same method should throw an exception if the parameter is not an array, so it does not fail silently (as the current approach also would).